runtime/lib/ttnn/runtime.cpp

```diff
@@ -50,6 +50,12 @@ Tensor createTensor(std::shared_ptr<void> data,
   return Tensor(tensor, data, DeviceRuntime::TTNN);
 }
 
+tt::target::DataType getTensorDataType(Tensor tensor) {
+  std::shared_ptr<::ttnn::Tensor> nnTensor =
+      std::static_pointer_cast<::ttnn::Tensor>(tensor.handle);
```

Comment:
Can we do `tensor.as<::ttnn::Tensor>` here?

Reply:
Thanks for the suggestion. updated.